fibonacci.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * A function to get Fibonacci value of given number
+ * @param number The input integer
+ * @return {number} Fibonacci value of `number`
+ * @example fibonacciValue(4) => 3 | fibonacciValue(6) => 8
+ * @see https://en.m.wikipedia.org/wiki/Fibonacci_number
+ * @author MohdFaisalBidda <https://github.com/MohdFaisalBidda>
+ */
+
+
+try {
```

Comment:
this `try`-`catch` block is misguided: the `catch` will never trigger; please remove the enclosing block